neofetch

```diff
@@ -242,7 +242,13 @@ get_model() {
             fi
         ;;
 
-        "Mac OS X") model="$(sysctl -n hw.model)" ;;
+        "Mac OS X") 
+            if [[ $(kextstat | grep "FakeSMC") != "" ]]; then
```

Comment:
Yep, that's the one. And yeah, double quotes (just checked, we use grep that way throughout neofetch so it's best to keep consistent)